src/components/PopupWithForm.js

```diff
@@ -7,12 +7,17 @@ export default class PopupWithForm extends Popup {
     this._handleFormSubmit = handleFormSubmit;
   }
 
+  setSaveButtonText(input) {
+    const saveButton = this._popupElement.querySelector(".modal__save-button");
+    saveButton.textContent = input;
+    return saveButton.textContent;
```

Comment:
Needs correction: This function should not return anything since it's a setter

Reply:
Done